### PR TITLE
documentation rescaling images

### PR DESCRIPTION
**Pull request by PanderMusubi**
See also https://github.com/gotify/server/issues/538

### CODE REVIEW

**Review by jmattheis — APPROVED** [review]
Thanks, I've already fixed my remarks.

File: docs/rescale-images.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Install the following packages:
+
+```bash
+sudo apt-get install imagemagick optipng
+```
+
+# Script
+
+Create a script, e.g. `/opt/gotify-rescale-images.sh` containing
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply docs/rescale-images.md]: Like the systemd file, I keep this file outside of /opt/gofity but in /opt.

File: docs/rescale-images.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+If users upload images with a transparent background that make the image hard to
+see, it is possible to enforce a white background by adding the following line
+before the `optipng` command for PNG images
+
+```shell
+            convert $FILE -background white -alpha remove -alpha off $FILE
+```
+
+and add it in an extra `if` for GIF images.
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply docs/rescale-images.md]: pkay

**Comment by PanderMusubi** [comment]
Thanks for merging

**Pull request merged**